.github/workflows/on_merge_to_main.yml

```diff
@@ -8,67 +8,17 @@ on:
       - main
 
 jobs:
-  prerelease:
-    name: Prerelease test and build
+  version_or_publish:
```

Comment:
Combining prerelease, and publish as it takes longer to upload the build assets than to build them with NX.